static/css/style.css

```diff
@@ -587,6 +587,13 @@ select {
 	border-bottom: 1px dotted #ebebeb;
 }
 
+.post__lead {
```

Comment:
```css
.post__lead {
	margin-top: 4px;
	margin-top: .25rem;
	margin-bottom: 0;
	font-size: 16px;
	font-size: 1rem;
	font-style: italic;
}
```

Reply:
In style.css [Line1199](https://github.com/Vimux/Mainroad/pull/63/files#diff-a16a51aaea7dc21d3202074fc7228024L1199) (between `.list__title` and `.list__meta`) add:

```css
.list__lead {
	font-size: 14px;
	font-size: .875rem;
}
```